doc/cli/operator-sdk_alpha_olm.md

```diff
@@ -15,7 +15,9 @@ Manage the Operator Lifecycle Manager installation in your cluster
 ### SEE ALSO
 
 * [operator-sdk alpha](operator-sdk_alpha.md)	 - Run an alpha subcommand
+* [operator-sdk alpha olm down](operator-sdk_alpha_olm_down.md)	 - Tears down the operator with Operator Lifecycle Manager
 * [operator-sdk alpha olm install](operator-sdk_alpha_olm_install.md)	 - Install Operator Lifecycle Manager in your cluster
```

Comment:
since it will enable the OLM on the cluster wdyt about we use to enable/disable and install/uninstall?

Reply:
The terminology will change depending on how you view "running software in a Kubernetes cluster". For example, OLM has two operator components that manage operator installation through CSVs, so technically you're "installing" both OLM and your operator. You're not explicitly "enabling" anything because installation implies enablement here. For now, OLM installation management will use `olm install/uninstall` and operator installation management will use `olm up/down`.

That being said, there's room to adjust command names while these commands are in `alpha`. We can discuss naming when we decide to move them from `alpha` to the stable CLI.